.github/workflows/e2e.yml

```
@@ -51,7 +51,7 @@ jobs:
     name: Cancel redundant jobs
     runs-on: ubuntu-latest
     timeout-minutes: 3
-    if: github.event_name != 'schedule'
+    if: ${{ github.ref != 'refs/heads/master' && github.event_name != 'schedule' }}
```

```suggestion
    if: github.ref != 'refs/heads/master' && github.event_name != 'schedule'
```

Pretty sure this works without the`${{` `}}`

---

:+1: for some reason I was thinking it'd be invalid YAML due to the && but I'm not sure why. I pushed another commit since committing the suggestion results in an unsigned commit.